service/src/test/java/uk/gov/hmcts/reform/fpl/config/utils/ElementUtilsTest.java

```diff
@@ -75,6 +76,35 @@ void shouldReturnUuidWhenMapRepresentationOfDynamicListPassed() {
         }
     }
 
+    @Nested
+    @ExtendWith(SpringExtension.class)
+    @ContextConfiguration(classes = {JacksonAutoConfiguration.class})
+    class DynamicListSelectedElement {
+        @Autowired
+        private ObjectMapper mapper;
```

Comment:
shouldn't need to autowire it in or have any of the spring stuff. just use objectmapper = new ObjectMapper()

Reply:
Updated the test